### ISSUE TITLE
CSS collision

### ISSUE THREAD

**Issue by awallat** [shlomiassaf/angular2-modal#228]
With version 2.0.1 I get a style conflict when I open a modal. This is probably because of the internal style of the component "message-modal.component.ts" with:
```
.form-group {
    margin-top: 10px;
}
```

The encapsulation is also set to "ViewEncapsulation.None" so that the style is applied to the whole application. Hence my forms are affected by this style.

I suggest to remove the encapsulation.

**Comment by iherbivore** [comment]
I agree. I don't know what drove the  "ViewEncapsulation.None" but I do see it causing problems. If it was only done to allow external style rules to be applied, the `/deep/` tag takes care of that.

**Comment by windmaomao** [comment]
this bug is serious. Because it'll populate external stylesheet.  At least, we need to get rid of 'ViewEncapsulation.None', instead using `::host`

**Comment by sumit-microsoft** [comment]
Was anyone able to get rid of this conflicting css anyhow ? any workarounds ?
